### PR TITLE
Groom changelog /1.14.0

### PR DESCRIPTION
**Pull request by alkwa-msft**
# What
Changelog refinement for our stable release

# Why
We want to prune the changelog so developers can understand what bug fixes, features, and improvements they will have access to with 1.14.0

It would be helpful for people to mainly review the stable changelog in communication-react packlet
packages/communication-react/CHANGELOG.stable.md

### CODE REVIEW

**Review by JamesBurnside — APPROVED** [review]

File: packages/communication-react/CHANGELOG.stable.md [hunk 1]
```diff
@@ -7,104 +7,45 @@ This log was last generated on Thu, 14 Mar 2024 03:50:48 GMT and should not be m
 ## [1.14.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.14.0)
```
Comment (mgamis-msft) [inline packages/communication-react/CHANGELOG.stable.md:7]: The link needs to be changed to https://github.com/azure/communication-ui-library/tree/1.14.0

File: packages/communication-react/CHANGELOG.stable.md [hunk 2]
```diff
@@ -7,104 +7,45 @@ This log was last generated on Thu, 14 Mar 2024 03:50:48 GMT and should not be m
...
-- Resolve issue where envHelper would return early on error throw when appsettings.json doesn't exist ([PR #4161](https://github.com/azure/communication-ui-library/pull/4161) by 73612854+palatter@users.noreply.github.com)
-- Revert "[Chat] Wrap ImageOverlay with theme provider and change some of the APIs (#4117)" ([PR #4131](https://github.com/azure/communication-ui-library/pull/4131) by 107075081+Leah-Xia-Microsoft@users.noreply.github.com)
 - Fix hanging express.close if have unclosed connection ([PR #4138](https://github.com/azure/communication-ui-library/pull/4138) by 77021369+jimchou-dev@users.noreply.github.com)
-- Remove CC in Storybook ([PR #4200](https://github.com/azure/communication-ui-library/pull/4200) by 107075081+Leah-Xia-Microsoft@users.noreply.github.com)
-- Fix crush when inline image is deleted from another client ([PR #4261](https://github.com/azure/communication-ui-library/pull/4261) by 107075081+Leah-Xia-Microsoft@users.noreply.github.com)
-
 
 ## [1.13.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.13.0)
```
Comment (mgamis-msft) [inline packages/communication-react/CHANGELOG.stable.md:50]: Might as well fix this link to be https://github.com/azure/communication-ui-library/tree/1.13.0 😅

**Review by ShaunaSong — APPROVED** [review]

**Review by Leah-Xia-Microsoft — APPROVED** [review]

**Review by carocao-msft — APPROVED** [review]

File: packages/communication-react/CHANGELOG.stable.md (hunk 2 above)
Comment (alkwa-msft) [reply packages/communication-react/CHANGELOG.stable.md]: done

File: packages/communication-react/CHANGELOG.stable.md (hunk 1 above)
Comment (alkwa-msft) [reply packages/communication-react/CHANGELOG.stable.md]: done

**Review by mgamis-msft — APPROVED** [review]

**Pull request merged**